Sources/ISDBTibs/TibsToolchain.swift

```diff
@@ -120,7 +120,7 @@ extension TibsToolchain {
 
     let envVar = "INDEXSTOREDB_TOOLCHAIN_PATH"
```

Comment:
Please rename `INDEXSTOREDB_TOOLCHAIN_PATH` to something like `INDEXSTOREDB_TOOLCHAIN_BIN_PATH`.  At least on Darwin, the terminology "toolchain path" means a path that does _not_ include the "/usr" component.

Reply:
Will do.

Reply:
Renamed the variable like you wanted.